assets/app/scripts/controllers/overview.js

```diff
@@ -180,10 +181,17 @@ angular.module('openshiftConsole')
 
       // Hide our deployer pods since it is obvious the deployment is
       // happening when the new deployment appears.
+
+      // Check for the deployer pod annotation.
       if (annotationFilter(pod, "openshift.io/deployment.name")) {
         return false;
       }
 
+      // Check for the deployer pod label.
+      if ($filter('label')(pod, "openshift.io/deployer-pod-for.name")) {
```

Comment:
I'm trying to avoid creating a variable called `labelFilter` to avoid confusion with `LabelFilter`. Suggestions?